src/content/roadmap/merge/issuance/index.md

```diff
@@ -101,31 +101,50 @@ On top of the fee burn implemented by the London upgrade, validators can also in
 
 ### Calculating average gas price for deflation {#calculating-average-gas-price-for-deflation}
 
-As discussed above, the amount of ETH issued in a given day is dependent upon the total ETH staked. At time of writing, this is approximately 1700 ETH/day.
+As discussed above, the amount of ETH issued in a given day is dependent upon the total ETH staked. At time of writing, this is approximately $1,700$ ETH/day.
 
-To determine the average gas price required to completely offset this issuance in a given 24-hour period, we'll start by calculating the total number of blocks in a day, given a block time of 12 seconds:
+To determine the average gas price required to completely offset this issuance in a given 24-hour period, we'll start by calculating the total number of blocks in a day, given a block time of $12\sec$:
 
-- `(1 block / 12 seconds) * (60 seconds/minute) = 5 blocks/minute`
-- `(5 blocks/minute) * (60 minutes/hour) = 300 blocks/hour`
-- `(300 blocks/hour) * (24 hours/day) = 7200 blocks/day`
+$$
+{1\space block \over 12\sec} \times 60 {\sec \over \min} = 5 {blocks \over \min} \\
+\downarrow \\
+5 {blocks \over \min} \times 60 {\min \over hour} = 300 {blocks \over hour} \\
+\downarrow \\
+300 {blocks \over hour} \times 24 {hours \over day} = 7200 {blocks \over day}
+$$
```

Comment:
This seems horrific for translations... Is there a simpler way to do this?

cc: @lukassim

Reply:
Hm... yeah.

Not sure I have a great solution here... but feel like I'd still lean towards trying to implement. @lukassim Do you have any idea if it'd be possible to block all text in these segments aside from things like a unit, ie "sec" "min" "day" "blocks", etc?

Reply:
Not really, we can hide individual strings, but not parts of strings.

Having taking a look at this in Crowdin, I think this would be very confusing for translators and can see a lot of issues and incorrect translations.

A good enough solution could be to implement this in English but not other languages, though we don't really have a good way of doing that...

<img width="553" alt="Screenshot 2023-07-20 at 16 12 54" src="https://github.com/ethereum/ethereum-org-website/assets/37338979/858a7b04-66de-473b-82bb-2b515e0fba75">